commands/curation/curationaudit.go

```diff
@@ -49,17 +50,50 @@ const (
 		"This package manager however isn't supported by this command."
 
 	TotalConcurrentRequests = 10
+
+	MinArtiMavenSupport = "7.82.0"
+	MinArtiXraySupport  = "3.92.0"
 )
 
 var CurationOutputFormats = []string{string(outFormat.Table), string(outFormat.Json)}
 
-var supportedTech = map[coreutils.Technology]func() (bool, error){
-	coreutils.Npm: func() (bool, error) { return true, nil },
-	coreutils.Maven: func() (bool, error) {
-		return clientutils.GetBoolEnvValue(utils.CurationMavenSupport, false)
+var supportedTech = map[coreutils.Technology]func(ca *CurationAuditCommand) (bool, error){
+	coreutils.Npm: func(ca *CurationAuditCommand) (bool, error) { return true, nil },
+	coreutils.Maven: func(ca *CurationAuditCommand) (bool, error) {
+		return ca.checkSupportByVersionOrEnv(coreutils.Maven, MinArtiMavenSupport, MinArtiXraySupport, utils.CurationMavenSupport)
 	},
 }
 
+func (ca *CurationAuditCommand) checkSupportByVersionOrEnv(tech coreutils.Technology, minRtVersion, minXrayVersion, envName string) (bool, error) {
+	flag, errResolveFlag := clientutils.GetBoolEnvValue(envName, false)
+	if errResolveFlag != nil {
+		log.Error(errResolveFlag)
+	}
+	if flag {
+		return true, nil
+	}
```

Comment:
```suggestion
	if flag, errResolveFlag := clientutils.GetBoolEnvValue(envName, false); errResolveFlag != nil {
		log.Error(errResolveFlag)
	} else if flag {
		return true, nil
	}
```
Why are we forcing the use of this using env even if Xray and Artifactory are not supporting it? what is the use-case and how it will work?

Reply:
this is for cases where you have milestones, snapshots version, mainly for development and demo environment